grype/presenter/format.go

```diff
@@ -9,10 +9,9 @@ const (
 	jsonFormat      format = "json"
 	tableFormat     format = "table"
 	cycloneDXFormat format = "cyclonedx"
+	cycloneDXXML    format = "cyclonedx-xml"
 	sarifFormat     format = "sarif"
 	templateFormat  format = "template"
-	embeddedVEXJSON format = "embedded-cyclonedx-vex-json"
```

Comment:
I think we should keep these names (allow users to still input them) and add a deprecation warning with some hints on the right version to use. We should also open an issue to remove these options and add it under a grype 1.0 milestone.

Reply:
Names have been kept and added as part of a deprecation scheme